README.md

```diff
@@ -22,22 +22,29 @@ Note: Due to [maps not being supported in protoc v2.X](https://github.com/google
 
 ### Go
 
-Code generation for Go has moved to the [Sonde-Go](https://github.com/cloudfoundry/sonde-go) library.
+The [Sonde-Go](https://github.com/cloudfoundry/sonde-go) library can be used instead.
 
-### Java
+Or, from the base directory, where `$DST_DIR` is the desired destination:
+```
+protoc -I=events --go_out=. events/*.proto
+mv github.com/cloudfoundry/dropsonde/events/*.pb.go $DST_DIR
+rm -rf github.com
+```
 
-1. Build the go code first (see above) so that all the imports are available
+### Java
 
-2. Generate the Java code (optionally providing a target path as a directory)
-   ```
-   ./generate-java.sh [TARGET_PATH]
-   ```
+From the base directory, where `$DST_DIR` is the desired destination:
+```
+protoc -I=events --java_out=. events/*.proto
```

Comment:
Forgot to include instructions for downloading pre-reqs to get to the `protoc` command.